lib/aur-fetch

```diff
@@ -8,88 +8,86 @@ readonly startdir=$PWD
 readonly PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 
 # default options
-mode=git recurse=0
+log=directory mode=git recurse=0
 
 git_stat_patch() {
     git --no-pager log -O "$1" --patch --stat '..@{upstream}'
 }
 
-merge_upstream() {
-    # set destination of patch file
-    local pkg=$1 log_dir=$2 output
-
-    if [[ -d $log_dir ]]; then
-        output=$log_dir/$pkg.patch
-    else
-        output=/dev/stdout
-    fi
-
-    # default to showing PKGBUILD diffs first (#399)
+upstream_sync() {
     local orderfile=${GIT_DIR:-$PWD/.git}/orderfile
 
+    # default to showing PKGBUILD in diffs first (#399)
     if [[ ! -s $orderfile ]]; then
         echo PKGBUILD >"$orderfile"
     fi
 
+    # get last upstream revision
     git fetch -v >&2
 
-    # only print log on upstream changes
     if [[ $(git rev-parse HEAD) != $(git rev-parse '@{upstream}') ]]; then
-        git_stat_patch "$orderfile" >> "$output"
+	# only print log on upstream changes
+	if [[ -f $1 ]]; then
```

Comment:
First argument is the pkgname, second argument is the log destination.
So this check should be done against `$2`, not `$1`

```suggestion
	if [[ -f $2 ]]; then
```